functions/api/v1/download-latest-installer/[arch].js

```diff
@@ -0,0 +1,42 @@
+const MAVEN = "https://maven.quiltmc.org/repository/release"
```

Comment:
Please run Prettier first to conform to style standards ;3

Reply:
This project doesn't have any prettier set up